src/script/descriptor.h

```diff
@@ -32,6 +32,9 @@ struct Descriptor {
     /** Whether the expansion of this descriptor depends on the position. */
     virtual bool IsRange() const = 0;
 
+    /** Whether this descriptor has all information about signing (ignoring private keys). */
+    virtual bool IsSolvable() const = 0;
```

Comment:
Curious where this might be used in the future, since right now it is only called by test code.

Reply:
For example an `descriptorprocesspsbt` RPC can complain if a non-solvable descriptor is passed (which is useless for its purpose).